source/features/parse-backticks.tsx

```diff
@@ -29,8 +29,9 @@ function repoInit(): void {
 		'.release-header', // `isReleasesOrTags` Headers
 		'[id^="check_suite"] a.link-gray-dark', // `isActions`
 		'.repository-content .pr-toolbar h2', // `isActions` run
-		'.Details[data-issue-and-pr-hovercards-enabled] .Details-content--hidden a:first-child', // `isRepoRoot`
+		'.Details[data-issue-and-pr-hovercards-enabled] .Details-content--hidden a.link-gray-dark', // `isRepoRoot`
 		'.Details[data-issue-and-pr-hovercards-enabled] .Details-content--hidden pre', // `isRepoRoot`
+		'.Details[data-issue-and-pr-hovercards-enabled] .d-none a.link-gray-dark', // `isRepoRoot`
```

Comment:
This is the commit title when the "Latest commit design updates" feature preview is enabled. I know it is not suitable to deal with preview features, but they hide the expand button in the "old" design, somehow forcing you to use the new design... So I assume this is wanted (an exception).